README.md

```diff
@@ -397,6 +397,8 @@ GRDB can be installed as a framework, or a static library.
 
 The [Swift Package Manager](https://swift.org/package-manager/) automates the distribution of Swift code. To use GRDB with SPM, add a dependency to `https://github.com/groue/GRDB.swift.git`
 
+GRDB two libraries, `GRDB` and `GRDB-dynamic`. Pick only one. When in doubt, prefer `GRDB`. The `GRDB-dynamic` library can reveal useful if you are going to link it with multiple targets within your app and only wish to link to a shared, dynamic framework once. See [#How to link as dynamic](https://forums.swift.org/t/how-to-link-a-swift-package-as-dynamic/32062)
```

Comment:
Excuse me, I suggested a typo 😅 

```suggestion
GRDB offers two libraries, `GRDB` and `GRDB-dynamic`. Pick only one. When in doubt, prefer `GRDB`. The `GRDB-dynamic` library can reveal useful if you are going to link it with multiple targets within your app and only wish to link to a shared, dynamic framework once. See [#How to link as dynamic](https://forums.swift.org/t/how-to-link-a-swift-package-as-dynamic/32062)
```

Reply:
You know I saw that, and thought "I've just come out of debating with the dude who worked on SwiftPM and now I'm going to debate Gwendal? I'm sure he wanted it to read that way" 😅